### ISSUE TITLE
Why can't I find my code after build?

### ISSUE THREAD

**Issue by qq253498229** [nwutils/nw-builder#806]
### Issue Type

- [X] Bug Fix
- [ ] Feature
- [ ] Other

### Current/Missing Behaviour

I used the 4.1.0 version of nw-builder to build my code.

After the build, when the application started, nwjs displays the default page. 

I looked at the file structure after build and found that app.nw is an empty 0kb file. At the same time, I also tried to build win64 application, package.nw is also 0kb.

What did I do wrong?

Here is my code:

<img width="1317" alt="image" src="https://user-images.githubusercontent.com/31200516/222034007-a4e8ca78-a95b-4026-a8e1-79924250b5a1.png">

dist/package.json:

```
{
  "name": "demo-nwjs1",
  "main": "index.html"
}
```

And I build with this command:

```
node build.mjs
```

And I also try other way, that add scripts in package.json, result still same:

```
"build1": "nwbuild ./dist/* --mode=build --version=0.73.0 --flavor=normal --platform=win --arch=x64 --outDir=./out --cacheDir=./cache",
```

### Expected/Proposed Behaviour

build with my code

### Additional Info

- Operating System: MacOS 13.2 (22D49)
- Node version: v18.14.2
- NW.js version: 0.73.0
- Repro link:
- ...

**Comment by tharatau** [comment]
Not sure how it happened, but `package.nw` got turned into a file!

**Comment by tharatau** [comment]
This has been fixed in `v4.1.0-beta.2`!
